image/src/main/java/edu/artic/image/GlideModule.kt

```diff
@@ -44,6 +54,25 @@ class GlideModule : AppGlideModule() {
 
     }
 
+    override fun registerComponents(context: Context, glide: Glide?, registry: Registry?) {
+        /**
+         * Configuration to use OkHttp client as the network client for Glide.
+         * OkHttp is also configured here to cache the image.
+         * Glide doesn't show the latest image if its from the same URL.
+         * OkHttp solves this problem by caching the image considering E-Tag and Last-Modified headers.
+         *
+         * @see https://github.com/bumptech/glide/issues/1847
+         */
+        val cacheSize: Long = 10 * 1024 * 1024 // 10 MiB
+        val clientBuilder = OkHttpClient.Builder()
+                .addInterceptor(HttpLoggingInterceptor().setLevel(HttpLoggingInterceptor.Level.HEADERS))
```

Comment:
Please do not enable logging in release builds.

Reply:
Now interceptor will only be added if build is `Debug`.